Source/Fuse.Models/Fuse.Models.unoproj

```diff
@@ -0,0 +1,33 @@
+{
+  "Copyright": "Copyright (c) Fusetools AS 2017",
+  "Description": "Native Location",
+  "Publisher": "Fusetools AS",
+  "Version": "1.4.0-rc4",
+  "Package": {
+    "ProjectUrl": "https://fusetools.com"
+  },
+  "Packages": [
+    "Uno.Collections",
+    "Uno.Threading",
+    "Uno.Permissions",
```

Comment:
I doubt that models requires Uno.Permissions... Also, are you sure it needs Uno.Threading?